src/addons/qbehaviour/deferredcbm/services/handlers/deferredcbm.ts

```diff
@@ -55,8 +55,9 @@ export class AddonQbehaviourDeferredCBMHandlerService implements CoreQuestionBeh
             question,
             componentId,
             siteId,
-            this.isCompleteResponse.bind(this),
-            this.isSameResponse.bind(this),
+            (question, answers, component, componentId) => this.isCompleteResponse(question, answers, component, componentId),
+            (question, prevAnswers, prevBasicAnswers, newAnswers, newBasicAnswers, component, componentId) =>
+                this.isSameResponse(question, prevAnswers, prevBasicAnswers, newAnswers, newBasicAnswers, component, componentId),
```

Comment:
```suggestion
            (...args) => this.isCompleteResponse(...args),
            (...args) => this.isSameResponse(...args),
```

In most places, I guess it's ok to add arguments explicitly because it's more readable. But if there are a lot of arguments, such as here, I think this approach is better. If the arguments are typed, as it happens here, this is still type safe.